<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchView</class>
 <widget class="QWidget" name="BatchView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>346</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QToolBar" name="toolBar">
     <property name="styleSheet">
      <string>QToolBar{border: none;}</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="rowButtonsLayout">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="toolTip">
        <string>The progress through processing the current reduction.</string>
       </property>
       <property name="whatsThis">
        <string>Shows the current progress when processing.</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="instrumentLabel">
       <property name="text">
        <string>Instrument:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="instrumentSelector">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select the instrument to assume for fetching runs</string>
       </property>
       <property name="whatsThis">
        <string>Specifies the instrument that data being processed was generated by. This is used to help identify the correct data to load when given sample run numbers to process.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="notebookCheckbox">
       <property name="toolTip">
        <string>Whether to generate a python notebook when performing a reduction</string>
       </property>
       <property name="text">
        <string>Output Notebook</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="processButton">
       <property name="toolTip">
        <string>Begin reducing the data in the table</string>
       </property>
       <property name="text">
        <string>Process</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/stat_rows.png</normaloff>:/stat_rows.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../widgets/common/MantidPlot/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
